app/views/duties/open_drop_modal.js.erb

```diff
@@ -0,0 +1,4 @@
+
+
+$('#modal-window1').html("<%= escape_javascript(render 'duties/open_drop_modal') %>");
```

Comment:
It would be best not to use `.js.erb` files. Can we somehow refactor this?

Reply:
I don't know how to. :(

Reply:
I'll help you on this one gimme a moment